controller/space_iterations_test.go

```diff
@@ -151,7 +149,7 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIterationWithOptionalValues
 		rootItr = &iteration.Iteration{
 			SpaceID:    testSpace.ID,
 			Name:       testSpace.Name,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c